Source/Particles/WarpXParticleContainer.H

```diff
@@ -382,7 +382,7 @@ protected:
     std::string physical_element;
 
     int do_resampling = 0;
-
+    bool m_rz_random_theta = false;
```

Comment:
Should we set the default to `true` instead, given that evenly spaced particles seemingly tend to produce wrong results?

Reply:
Good question, I would say yet let's default it to `true`. @RemiLehe any opinion?

Reply:
Yes, I agree that the default should be true. Without this, we can sometimes observe artifacts due to the fact that particles are align along certain axes in the transverse plane.

Reply:
Alright, thanks! I changed it, so the PR is ready for review... but I am surprised that no test failed although we changed this injection default.